features/mbedtls/platform/inc/platform_mbed.h

```diff
@@ -17,6 +17,12 @@
  *  This file is part of mbed TLS (https://tls.mbed.org)
  */
 
+
+#ifndef __PLATFORM_MBED__H__
+#define __PLATFORM_MBED__H__
+
+#include "default_random_seed.h"
```

Comment:
This `default_random_seed.h` seems to be `TARGET_PSA` specific. How will the compiler find it when the target is not `TARGET_PSA`?

Reply:
fixed